api/lib/getJSON.js

```diff
@@ -1,5 +1,27 @@
+const object = require('lodash/fp/object');
+
+const reduceObj = (accumulator, currentObj) => {
```

Comment:
eventually would be nice to, instead of repeating this code across 3 codebases, maybe pull into a seperate lib?

Reply:
if it keeps growing we will.. right now the lib dirs are too different